### PR TITLE
added pet item

### PR DESCRIPTION
**Pull request by Stekerser**

### CODE REVIEW

File: constants/parents.json [hunk 1]
```diff
@@ -1256,6 +1256,7 @@
     "PET_ITEM_FISHING_SKILL_BOOST_COMMON"
   ],
   "PET_ITEM_FARMING_SKILL_BOOST_EPIC":[
+    "PET_ITEM_FARMING_SKILL_BOOST_UNCOMMON",
```
Comment (jani270) [inline constants/parents.json:1259]: ????

**Review by jani270 — CHANGES_REQUESTED** [review]

**Comment by jani270** [comment]
and where do you get that item

**Comment by Stekerser** [comment]
quest on farming island

**Review by jani270 — APPROVED** [review]

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**